ci/build.sh

```diff
@@ -0,0 +1,2 @@
+# export PATH=$HOME/.local/bin:$CACHE_DIR/z3/bin:$CACHE_DIR/bin:$PATH
```

Comment:
Do we need this commented out line?

Reply:
In theory not. I moved it to an external step in SemaphoreCI because *test.sh* and *test_long.sh* also need them, so I wanted to keep this line as a reminder in case we move again. Do you think it should go?

Reply:
I would say so. I'd rather have no commented out code. But it's a matter of personal taste.